extensions/spark/kyuubi-extension-spark-3-5/src/test/scala/org/apache/spark/sql/ZorderSuiteBase.scala

```diff
@@ -332,19 +340,20 @@ trait ZorderSuiteBase extends KyuubiSparkSQLExtensionTest with ExpressionEvalHel
         assert(df1.queryExecution.analyzed.isInstanceOf[InsertIntoHadoopFsRelationCommand])
         checkSort(df1.queryExecution.analyzed.children.head)
 
-        withListener(
-          s"""
-             |CREATE TABLE zorder_t4 USING PARQUET
-             |TBLPROPERTIES (
-             | 'kyuubi.zorder.enabled' = '$enabled',
-             | 'kyuubi.zorder.cols' = '$cols')
-             |
-             |SELECT $repartition * FROM
-             |VALUES(1,'a',2,4D),(2,'b',3,6D) AS t(c1 ,c2 , c3, c4)
-             |""".stripMargin) { write =>
-          assert(write.isInstanceOf[InsertIntoHadoopFsRelationCommand])
-          checkSort(write.query)
-        }
+        // TODO: CreateDataSourceTableAsSelectCommand is not supported
```

Comment:
`catalogTable` is empty

https://github.com/apache/kyuubi/blob/master/extensions/spark/kyuubi-extension-spark-3-5/src/main/scala/org/apache/kyuubi/sql/zorder/InsertZorderBeforeWriting.scala#L151C40-L151C68

Reply:
why does `CreateDataSourceTableAsSelectCommand`not support ? CreateDataSourceTableAsSelectCommand should call InsertIntoHadoopFsRelationCommand internally.

Reply:
InsertIntoHadoopFsRelationCommand's catalogTable will be empty

Reply:
https://github.com/apache/spark/blob/4771638b3a6873e329bb3bdde5abeeeac059acb8/sql/core/src/main/scala/org/apache/spark/sql/execution/command/createDataSourceTables.scala#L185-L186